### ISSUE TITLE
Infinity Filter Import/Export Error on close

### ISSUE THREAD

**Issue by fkloiber** [raiguard/EditorExtensions#183]
### Description

Closing the dialog to import/export inventory infinity filters via the back button results in the following error:

```
The mod Editor Extensions (2.3.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event EditorExtensions::on_gui_click (ID 1)
Invalid type: 3.
stack traceback:
	[C]: in function '__newindex'
	__EditorExtensions__/scripts/inventory-filters.lua:101: in function 'handler'
	__flib__/gui.lua:177: in function 'handler'
	__core__/lualib/event_handler.lua:47: in function <__core__/lualib/event_handler.lua:45>
```

When closing the dialog via clicking off it, an error instead occurs when closing the inventory:

```
The mod Editor Extensions (2.3.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event EditorExtensions::on_gui_closed (ID 96)
Invalid type: 3.
stack traceback:
	[C]: in function '__newindex'
	__EditorExtensions__/scripts/inventory-filters.lua:219: in function 'handler'
	__core__/lualib/event_handler.lua:47: in function <__core__/lualib/event_handler.lua:45>
```

Relevant versions:
* Factorio / Space Age mods 2.0.11
* Editor Extensions 2.3.0
* flib 0.15.0

### Reproduction

Two Variations:

1\. Start new game
2. Enter Editor
3. Click on button "Import infinity filters" or "Export infinity filters"
4. Try to close the dialog via the Back button

1\. Points 1-3 as above
2. Click off the dialog to close it
3. Close the inventory

**Comment by grossws** [comment]
See #180, its upstream bug

**Comment by grossws** [comment]
Fixed in Factorio 2.0.12
